src/assets/styles/global.scss

```diff
@@ -47,20 +34,39 @@
   --calcite-scrim-background: #{rgba($blk-240, 0.75)};
 }
 
+:root {
+  @extend %type-vars;
+  @include calcite-theme-light-extended();
+
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+
+  --calcite-border-radius: 3px;
+  --calcite-ui-opacity-disabled: 0.5;
+  --calcite-panel-width-multiplier: 1;
+
+  @apply font-sans;
+}
+
+.calcite-theme--dark,
+[theme="dark"] {
```

Comment:
☝️  Are we keeping this here for backwards compatibility so that this doesn't introduce a breaking change?

Reply:
Yes. we are.